embedded-influxdb/src/main/java/com/playtika/test/influxdb/InfluxDBProperties.java

```diff
@@ -1,25 +1,26 @@
 package com.playtika.test.influxdb;
 
-import com.playtika.test.common.properties.CommonContainerProperties;
 import lombok.Data;
 import lombok.EqualsAndHashCode;
 import org.springframework.boot.context.properties.ConfigurationProperties;
 
+import com.playtika.test.common.properties.CommonContainerProperties;
+
 @Data
 @EqualsAndHashCode(callSuper = true)
 @ConfigurationProperties("embedded.influxdb")
 public class InfluxDBProperties extends CommonContainerProperties {
 
     static final String EMBEDDED_INFLUX_DB = "embeddedInfluxDB";
-    String dockerImage = "influxdb:1.5-alpine";
```

Comment:
we can not just delete the property

Reply:
Reverted properties changed. Updated code in order to support passing sutom dockerImageName as a parameter